contributors/README.md

```diff
@@ -111,6 +111,12 @@ See [Code Comment Style](./code-comment-style.md) for details.
 
 See [Commit Message and Pull Request Style](./commit-message-pr-style.md) for details.
 
+### Diagram Style
```

Comment:
Please also add it to the beginning `## Contents` section in this file.

Reply:
Added.

Reply:
> Please also add it to the beginning `## Contents` section in this file.

Added.